app/admin/ministry.rb

```diff
@@ -34,7 +33,7 @@
     f.semantic_errors
     f.inputs do
       f.input :name
-      f.input :description, as: :ckeditor
```

Comment:
Thanks for confirming. In my quick skim I had just seen the attribute 'description' and assumed it was one and the same.   Thank you for confirming.